### PR TITLE
docs(user-flows): add instructions for DevTools

### PR DESCRIPTION
**Pull request by adamraine**

### CODE REVIEW

File: docs/user-flows.md [hunk 1]
```diff
@@ -188,10 +248,9 @@ When mapping a user flow onto the Lighthouse modes, strive for each report to ha
 
 The below example codifies a user flow for an ecommerce site where the user navigates to the homepage, searches for a product, and clicks on the detail link.
 
-![Lighthouse User Flows Diagram](https://user-images.githubusercontent.com/2301202/135164371-20cc5c8c-e876-467f-985c-f85683afa8ee.png)
-
+![Lighthouse User Flows Diagram](https://user-images.githubusercontent.com/6752989/168678568-69aaa82f-0459-4c2a-8f46-467d7f06d237.png)
```
Comment (adamraine) [inline docs/user-flows.md:251]: Drive-by this diagram is unreadable in dark mode

**Review by paulirish — APPROVED** [review]
Thank you sir! I'm gonna land this and put up a PR with a bunch of changes.. will def be faster than trying to itemize it all here.
